index.html

```diff
@@ -235,23 +246,27 @@ <h4 id="about-heading">Let's Connect</h4>
                                 <img src="asset/chevron-down.png" alt="arrow icon">
                             </button>
                         </li>
-                        <li class="language-item">
-                            <div>
-                                <img src="asset/js-icon.png" alt="js icon">
-                                <p>Javascript</p>
-                            </div>
-                        </li>
-                        <li class="language-item">
-                            <div>
-                                <img src="asset/html-icon.png" alt="html icon">
-                                <p>HTML</p>
-                            </div>
-                        </li>
-                        <li class="language-item">
-                            <div>
-                                <img src="asset/css-icon.png" alt="css icon">
-                                <p>CSS</p>
-                            </div>
+                        <li>
+                            <ul id="language-list">
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/js-icon.png" alt="js icon">
+                                        <p>Javascript</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/html-icon.png" alt="html icon">
+                                        <p>HTML</p>
+                                    </div>
+                                </li>
+                                <li class="language-item">
+                                    <div>
+                                        <img src="asset/css-icon.png" alt="css icon">
+                                        <p>CSS</p>
+                                    </div>
+                                </li>
+                            </ul>
```

Comment:
This part of your application is not matching the Figma template of your choosing. The icons are smaller in the Figma template placed in the top left corner. The box containing the image and text should have a `border-radius: 8px` which is missing in your case. Kindly fix these issues.

| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/220935159-550ed597-905d-475d-8233-1eb3d72a7253.png) | ![image](https://user-images.githubusercontent.com/33268927/220935334-069ce6b4-f7f1-412a-9c28-6e9edd684bf4.png) |